bindata/network/openshift-sdn/000-ns.yaml

```diff
@@ -9,3 +9,16 @@ metadata:
   annotations:
     openshift.io/node-selector: "" #override default node selector
     openshift.io/description: "OpenShift SDN components"
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  # NOTE: this namespace is the portal for host network traffic
+  name: openshift-host-network
+  labels:
+    openshift.io/run-level: "0"
+    openshift.io/cluster-monitoring: "true"
+    network.openshift.io/policy-group: "host-network"
+  annotations:
+    openshift.io/node-selector: ""
```

Comment:
don't need that either; there will be no pods in this namespace, so we don't need to override node restrictions on it

In fact there was discussion somewhere about maybe we should apply a quota to this namespace to ensure that pods _can't_ be created in it, and I think that's a good idea. I don't know how to do that though... (If it requires creating a second object though, then maybe we should split the host-network objects into their own bindata subdir?)